eligibility_api/client.py

```diff
@@ -29,7 +29,7 @@ class RequestToken:
     def __init__(
         self,
         types: Iterable[str],
-        agency_id: str,
+        agency_identifier: str,
```

Comment:
I actually think we can simplify down to `agency` given that is what it is called in the spec: https://docs.calitp.org/eligibility-api/specification/#request